### PR TITLE
Promote image automation to stable and update roadmap

### PR DESCRIPTION
**Pull request by stefanprodan**
The image automation APIs have been promoted from `v1alpha2` to `v1beta1`. This PR changes the API version in the image automation guides and marks the image controllers and their APIs as production ready.

The GA roadmap has been updated with conformance testing, API consolidation and multi-tenancy tasks.

⚠️ DO NOT MERGE! This PR should be merged right after flux2 0.16.0 release.

### CODE REVIEW

File: content/en/docs/roadmap.md [hunk 1]
```diff
@@ -9,40 +9,54 @@ weight: 90
...
+    - [x] Review the spec of `ImageRepository`, `ImagePolicy` and `ImageUpdateAutomation`
+    - [x] Promote the image automation APIs to `v1beta1`
+
+- Conformance testing
+    - [x] End-to-end testing for Flux bootstrap on AMD64 and ARM64 clusters
+    - [ ] End-to-end testing for Flux image automation
+
+- API consolidation
+    - [ ] Adopt Kubernetes [kstatus](https://github.com/kubernetes-sigs/cli-utils/tree/v0.25.0/pkg/kstatus#conditions) standard conditions
+    - [ ] Standardize events metadata
```
Comment (hiddeco) [inline content/en/docs/roadmap.md:47]:
```suggestion
    - [ ] Standardize events and conditions metadata
```

would be a bit more accurate I think, as there are some planned changes in condition reasons and minor other condition related changes ahead, that are unrelated to just `kstatus`.

Comment (stefanprodan) [reply content/en/docs/roadmap.md:47]: Do you think the usual Flux user understands what `conditions` means in this context?

Comment (hiddeco) [reply content/en/docs/roadmap.md:47]: status conditions?

**Review by hiddeco — APPROVED** [review]
:checkered_flag: looks good to me, thanks Stefan
